### ISSUE TITLE
[FEATURE] Show name of the spell to be cast

### ISSUE THREAD

**Issue by Dobbelklick** [Hekili/hekili#4253]
### Before You Begin

- [X] I confirm that I have downloaded the latest version of the addon.
- [X] I am not playing on a private server.
- [X] I checked for an [existing, open ticket](https://github.com/Hekili/hekili/labels/enhancement) for this request and was not able to find one.
- [X] I edited the title of this feature request (above) so that it describes the issue I am reporting.

### Feature Request

I'd like to see the name of the skill that I'm getting recommended.

Like this:
![image](https://github.com/user-attachments/assets/78b39858-1ca1-4104-8de9-67fb9048038a)

### Additional Information

I think it would facilitate learning a class since you'd be able to parse the talent descriptions much faster.

**Comment by syrifgit** [comment]
Could be useful, not sure if this is more suited to a weakaura or not. There also exists already the functionality to pause the addon and see what it's recommending. `alt-shift-p` by default.

**Comment by johnnylam88** [comment]
I think it's simpler as a companion WeakAura instead of creating a separate text display for it. I can't imagine most people will use it.

**Comment by Hekili** [comment]
It might be viable to make this an option for the existing Caption feature.

**Comment by syrifgit** [comment]
It would certainly cut down on tickets, discord questions and reddit posts if there was a small, minimalist tooltip (mouseover) even without pausing that said only the name. It's still absolutely the # 1 FAQ. 

Captions might be pretty intrusive?

**Comment by johnnylam88** [comment]
I like mouseover tooltip because that's the instinctive thing to do as a user -- see icon, mouseover to see what it is.

**Comment by johnnylam88** [comment]
Another reason for the tooltip is that you can just show Blizzard's tooltip for the spell, which is automatically localized for the user's WoW client.

**Comment by Mabellah** [comment]
I agree with Dobbelklick. I'd like to see the name of the spell without having to mouse over.
